docs/make.jl

```diff
@@ -2,7 +2,7 @@ push!(LOAD_PATH,"../src/")
 using Documenter, JToric, DocumenterMarkdown
 
 # ensure JToric is loaded for the doc tests
-DocMeta.setdocmeta!(JToric, :DocTestSetup, :(using JToric, Random); recursive = true)
+DocMeta.setdocmeta!(JToric, :DocTestSetup, :(using JToric, Random, Oscar); recursive = true)
```

Comment:
This change was nonsense and needs to be reverted, or perhaps changed like this:
```suggestion
DocMeta.setdocmeta!(JToric, :DocTestSetup, :(using JToric, Random; using JToric.Oscar); recursive = true)
```